source/develop/troubleshooting-nfs-storage-issues.md

```diff
@@ -147,6 +147,37 @@ A new **nfs-check** script is now available to test whether an NFS export is rea
       # systemctl start rpcbind 
       # systemctl start nfs-server
 
+#### Synology DSM 7
+Synology DSM 7 adds / extends Access Control Lists (ACLs) on all shared folders, and removes `groupadd` and `useradd`...  
+To allow ovirt nodes to connect via NFS the ACL permissions and /etc/exports file need to be updated manually (*take backups before editing!*).
+
+1. Create the shared folder required as usual and allow you oVirt hosts IP's in the NFS permissions section
+
+2. Login to you Synology NAS via SSH (search the web for 'synology enable ssh access') as root or sudo to root once logged in
+
+3. Carefully add kvm (id 36) to /etc/group and vdsm (id 36) to /etc/passwd and /etc/shadow, if you need an encrypted password the python command below will create one
+
+        # python3 -c 'import crypt,getpass;pw=getpass.getpass();print(crypt.crypt(pw) if (pw==getpass.getpass("Confirm: ")) else exit())'
+
+4. Update the Synology ACLs to allow access to kvm:vdsM
+
+        # EXPORT_DIR=/volumeX/...
+
+        # synoacltool -set-owner "$EXPORT_DIR" group kvm:allow:rwxpdDaARWcCo:fd--
+        # synoacltool -add "$EXPORT_DIR" user:vdsm:allow:rwxpdDaARWcCo:fd--
+        # synoacltool -add "$EXPORT_DIR" group:kvm:allow:rwxpdDaARWcCo:fd--
+
+5.  Edit /etc/exports and replace the anonuid and anonguid values with 36, e.g.
+
+        /volume1/ovirt 1.1.1.1(rw,async,no_wdelay,crossmnt,insecure,no_root_squash,insecure_locks,sec=sys,anonuid=1025,anongid=100) 1.1.1.2(rw,async,no_wdelay,crossmnt,insecure,no_root_squash,insecure_locks,sec=sys,anonuid=1025,anongid=100)
+
+Would become
+
+        /volume1/ovirt 1.1.1.1(rw,async,no_wdelay,crossmnt,insecure,no_root_squash,insecure_locks,sec=sys,anonuid=36,anongid=36) 1.1.1.2(rw,async,no_wdelay,crossmnt,insecure,no_root_squash,insecure_locks,sec=sys,anonuid=36,anongid=36)
```

Comment:
Why not show simpler configuration in the examples?

    /volume1/ovirt *(rw,async,crossmnt,insecure,no_root_squash,insecure_locks,sec=sys,anonuid=36,anongid=36)

no_wdelay does not have any effect when async is used. Do we good reason to use crossmnt,insecure (with sec=sys) and no_root_squash with Synology DSM 7?

Reply:
Those are the options set by Synology, the only change I'm implying here is to the anonuid / anongid